docs/pipeline_config_documentation/pipeline_ingestion_yml.md

```diff
@@ -213,13 +217,17 @@ The Quality Control (QC) is run for each modality independently.
         If use_thr (previous parameter) is set to True, the threshold specified here will be used to define doublets.
     
 
-### RNA QC    
-This file section specifies parameters used for running Quality Control (QC) on the data.
-Note that we do not filter out cells or genes, as this is part of the subsequent workflow (preprocess).
+### RNA modality Quality Control
+In the following, we specify parameters used for running QC on the RNA modality data.
+In the ingestion workflow we compute cell and genes QC metrics (such as % of mitochondrial genes, number of genes expressed in a cell etc.) but we do not apply filtering, as this is part of the subsequent workflow (preprocess).
 Feel free to leave options blank to run with default parameters.
 
+#### Providing a gene list and defining actions on the genes
```

Comment:
I think this and the following subsections are quite unclear - at least I find them hard to understand. Maybe instead of having the three subsection that we have currently, we could just have two:
- Providing a gene list
- Defining actions on the genes (this section then contains everything from _Next, we define "actions" on the genes as follows:_ onwards, including the sections _Custom genes actions_ and _Cell cycle actions_.

What do you think about that?

Reply:
yes it makes sense and matches the description in the documentation of the custom genes lists (when i get to finally push my latest version!!!)